packages/forms/demo/text.stories.js

```diff
@@ -0,0 +1,379 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import '../dist/text.css';
+import React from 'react';
+
+export default {
+  title: 'Components/Forms/Text'
+};
+
+export const Default = {
+  /* <div className="col u-mt">
+    <div className="c-txt">
+      <label className={`c-txt__label ${isCompact && 'c-txt__label--sm'}`} htmlFor="media">
+        .c-txt__label
+      </label>
+      {hasHint && (
+        <span className={`c-txt__hint ${isCompact && 'c-txt__hint--sm'}`}>.c-txt__hint</span>
+      )}
+      <div
+        className={`c-txt__input c-txt__input--media ${isCompact && 'c-txt__input--sm'} ${
+          isBare && 'c-txt__input--bare'
+        } c-txt__input--${validation} ${state && state.join(' ')}`}
+      >
+        <div className="c-txt__input--media__figure">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="16"
+            height="16"
+            focusable="false"
+            viewBox="0 0 16 16"
+          >
+            <rect
+              width="15"
+              height="11"
+              x=".5"
+              y="2.5"
+              fill="none"
+              stroke="currentColor"
+              rx=".5"
+              ry=".5"
+            ></rect>
+            <path fill="currentColor" d="M.5 5h15v2H.5z"></path>
+          </svg>
+        </div>
+        <input
+          className="c-txt__input c-txt__input--bare c-txt__input--media__body"
+          disabled={state && state.includes('is-disabled')}
+          id="media"
+          placeholder=".c-txt__input.c-txt__input--media"
+          type="text"
+        />
+        <div className="c-txt__input--media__figure">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="16"
+            height="16"
+            focusable="false"
+            viewBox="0 0 16 16"
+          >
+            <path
+              fill="none"
+              stroke="currentColor"
+              stroke-linejoin="round"
+              d="M4.5 2.5H2a.5.5 0 0 0-.5.5v12a.5.5 0 0 0 .5.5h12a.5.5 0 0 0 .5-.5V3a.5.5 0 0 0-.5-.5h-2.5m-6.95 2C4.25 2.42 5.883.5 8 .5s3.75 1.92 3.45 4h-6.9z"
+            ></path>
+          </svg>
+        </div>
+      </div>
+      {hasMessage && (
+        <small className={`c-txt__message c-txt__message--${validation}`}>
+          .c-txt__message
+        </small>
+      )}
+    </div>
+  </div> */
```

Comment:
Leftover from the old story file, not sure if we still need this.

Reply:
Def not needed. Let's nuke this dead code.